pipeline/relation_embedder/relation_embedder/docker-compose.yml

```diff
@@ -1,5 +1,24 @@
-version: '3.3'
+# Run locally using `docker compose build lambda && docker compose run --rm  --service-ports lambda`
 services:
+  lambda:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: lambda_rie
+    volumes:
+      - ~/.aws:/root/.aws
+    ports:
+      - "9000:8080"
+    environment:
+      - AWS_PROFILE=platform-developer
+      - metrics_namespace=catalogue-dev_relation_embedder
+      - affected_works_scroll_size=50
+      - complete_tree_scroll_size=800
+      - index_batch_size=100
+      - index_flush_interval_seconds=60
+    env_file:
+      - .env
```

Comment:
The `.env` file contains the remaining pipeline specific variables, we could have a script to generate a file like this:

```
es_apikey=secret:elasticsearch/pipeline_storage_2024-11-18/relation_embedder/api_key
es_host=secret:elasticsearch/pipeline_storage_2024-11-18/public_host
es_denormalised_index=works-denormalised-2024-11-18
es_merged_index=works-merged-2024-11-18
es_protocol=https
es_port=443
```

Reply:
Script added, as part of `run_local.sh`